src/Avataaars/Random.elm

```diff
@@ -0,0 +1,304 @@
+            )
+
+
+generateHairColor : Generator HairColor
+generateHairColor =
+    Random.uniform Avataaars.HairColor.auburn
+        [ Avataaars.HairColor.black
+        , Avataaars.HairColor.blonde
+        , Avataaars.HairColor.blondeGolden
+        , Avataaars.HairColor.brown
+        , Avataaars.HairColor.brownDark
+        , Avataaars.HairColor.pastelPink
+        , Avataaars.HairColor.platinum
+        , Avataaars.HairColor.red
+        , Avataaars.HairColor.silverGray
+        ]
+
+
+generateHatColor : Generator HatColor
+generateHatColor =
+    Random.uniform Avataaars.HatColor.black
+        [ Avataaars.HatColor.blue01
+        , Avataaars.HatColor.blue02
+        , Avataaars.HatColor.blue03
+        , Avataaars.HatColor.gray01
+        , Avataaars.HatColor.gray02
+        , Avataaars.HatColor.heather
+        , Avataaars.HatColor.pastelBlue
+        , Avataaars.HatColor.pastelGreen
+        , Avataaars.HatColor.pastelOrange
+        , Avataaars.HatColor.pastelRed
+        , Avataaars.HatColor.pastelYellow
+        , Avataaars.HatColor.pink
+        , Avataaars.HatColor.red
+        , Avataaars.HatColor.white
+        ]
+
+
+generateSkinTone : Generator SkinTone
+generateSkinTone =
+    Random.uniform Avataaars.SkinTone.black
+        [ Avataaars.SkinTone.brown
+        , Avataaars.SkinTone.darkBrown
+        , Avataaars.SkinTone.light
+        , Avataaars.SkinTone.pale
+        , Avataaars.SkinTone.tanned
+        , Avataaars.SkinTone.yellow
+        ]
+
+
+generateTop : Generator Top
+generateTop =
+    Random.Extra.map6
+        (\topFacialHair topHatColorAccessory topAccessoryFacialHair topHatColorAccessoryFacialHair topHairColorAccessoryFacialHair facialHair hatColor accessory hairColor ->
+            Random.uniform (Avataaars.Top.TopFacialHair topFacialHair facialHair)
+                [ Avataaars.Top.TopHatColorAccessory topHatColorAccessory hatColor accessory
+                , Avataaars.Top.TopAccessoryFacialHair topAccessoryFacialHair accessory facialHair
+                , Avataaars.Top.TopHatColorAccessoryFacialHair topHatColorAccessoryFacialHair hatColor accessory facialHair
+                , Avataaars.Top.TopHairColorAccessoryFacialHair topHairColorAccessoryFacialHair hairColor accessory facialHair
+                ]
+        )
+        generateTopFacialHair
+        generateTopHatColorAccessory
+        generateTopAccessoryFacialHair
+        generateTopHatColorAccessoryFacialHair
+        generateTopHairColorAccessoryFacialHair
+        generateFacialHair
+        |> Random.Extra.andMap generateHatColor
+        |> Random.Extra.andMap generateAccessory
+        |> Random.Extra.andMap generateHairColor
+        |> Random.andThen identity
+
+
+generateTopFacialHair : Generator Avataaars.Top.TopFacialHair
+generateTopFacialHair =
+    Random.constant Avataaars.Top.Eyepatch
+
+
+generateTopHatColorAccessory : Generator TopHatColorAccessory
+generateTopHatColorAccessory =
+    Random.constant Avataaars.Top.Hijab
```

Comment:
Since there is only one option in this case, it can be a constant until more variants are added.